x-pack/metricbeat/module/aws/vpn/_meta/fields.yml

```diff
@@ -4,3 +4,15 @@
     `vpn` contains the metrics from Cloudwatch to track usage of VPN related resources.
   release: beta
   fields:
+    - name: metrics
+      type: group
+      fields:
+        - name: TunnelState.avg
+          type: integer
+          description: The state of the tunnel. For static VPNs, 0 indicates DOWN and 1 indicates UP. For BGP VPNs, 1 indicates ESTABLISHED and 0 is used for all other states.
+        - name: TunnelDataIn.sum
+          type: double
```

Comment:
Can `sum`'s be of `double` type?

Reply:
Yeah I think so because TunnelDataIn itself is double. So the sum of double is also a double 🤔

Reply:
👍